web/modules/custom/usa_admin_styles/benefitsCategoryLanguageFilter.js

```diff
@@ -0,0 +1,64 @@
+/*
+Terms in the "Benefits Category" taxonomy can be English or Español, and
+the "Benefits Category Search" field displayed while editing a Basic Page
+content type shows all the items by default.
+
+This script filters those categories by the content's selected language.
+
+This script assumes that each category includes its language at the end of the <label> element
+*/
+
+(function (jQuery, Drupal, drupalSettings) {
+
+    storeLanguageInDataAttribute(jQuery);
```

Comment:
You can add the data attribute via rewriting the field in the view.

1. Hide the original language field and move it to first

![image](https://github.com/usagov/usagov-2021/assets/493854/94283a73-d377-44c9-b231-5f44b78bd084)

2. Update the title field, under style settings you can uncheck the "add default classes" option. And under "Rewrite classes" you can put the markup you want. The replacement patterns section shows you what's available. Each field can see itself and any fields that came before it.

![image](https://github.com/usagov/usagov-2021/assets/493854/f2960711-f693-4505-abd9-908cf0b91c08)

3. The edit form's checkboxes should then have the data attribute:

![image](https://github.com/usagov/usagov-2021/assets/493854/ce5b2672-03f9-40f9-822b-057576692e0c)

4. Update `filterCategoriesByLanguage()` to find first span in each option with a `data-language` attribute and use it:

```js
if(jQuery(this).find('span[data-language]').first().attr( "data-language" ) != selectedLanguage){
```

Reply:
Then you don't need `storeLanguageInDataAttribute()`

Reply:
Thank you! I knew this should be possible. I think I hadn't tried changing the order of the fields... I added this change and it looks much cleaner this way.